test/tools/jsontests/StateTests.cpp

```diff
@@ -104,10 +104,6 @@ class generaltestfixture
 	generaltestfixture()
 	{
 		string casename = boost::unit_test::framework::current_test_case().p_name;
-		if (casename == "stBoundsTest" && !test::Options::get().memory)
-			return;
-		if (casename == "stMemoryStressTest" && !test::Options::get().memory)
-			return;
```

Comment:
Are you aware that `stMemoryStressTest` still exists at https://github.com/ethereum/cpp-ethereum/pull/4183/files#diff-60b4496c08307cc76ac346ca15cf03ecR180 ?